packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -84,9 +100,21 @@ class _MyHomePageState extends State<MyHomePage> {
         title: new Text(config.title),
       ),
       body: new Center(
+{{#withPluginHook}}
+        child: new Column(
```

Comment:
What I mean is that right now the main.dart.tmpl file is effectively forked into four versions:
* the one generated by `flutter create`
* the one generated by `flutter create --driver` or some such
* the one generated by `flutter create --plugin` or some such
* the one hypothetically generated by `flutter create --plugin --driver` or some such (I suspect we don't ever generate this today, I haven't checked)

We have carefully crafted the one generated by `flutter create` based on usability feedback to optimise for the first hour experience. The other versions haven't been tested and I'm worried about what will happen as the needs for these various versions diverge.